### DISCUSSION TITLE
What is the audio format that is initially downloaded?

### DISCUSSION THREAD

**Discussion by culschmid** [zotify-dev/zotify#30] [Q&A]
Trying to avoid unnecessary back and forth encoding if I can use whatever comes from the source.

**Comment by soucarlosxavier** [comment]
Hey, I've been using Zotify for the past 2 days and I love it. The default format of download is .ogg (for some reason), but you can either edit the config file (config.json) or define it in the command line (see README.md for guidance on how to do it). I've change the config file from .ogg to .mp3. 
Hope it helps

**Reply by FizzyMUC** [reply]
Hey, I tried the same but even though my config now says mp3, files are still downloaded as ogg. Any ideas?

**Reply by aari38** [reply]
you can change it by adding arguments to the command, i.e.
`zotify 'your Spotify link' --audio-format mp3 --ffmpeg-args '-b:a 320k'` to force it to transcode the .ogg files to 320kbps mp3 files after download of each song.
